### DISCUSSION TITLE
Dashboard Headers Overrun by charts.

### DISCUSSION THREAD

**Discussion by specialagentsoftware** [datalust/seq-tickets#1951] [Help]
Hello all, We installed Version 2023.4.10097 in our dev environment to correct an issue we previously had reported. We noticed after the install that we had headers that were overrunning the charts on dashboards. I checked in both our environments and in both versions of Seq that we have installed (PROD : 2022.1.8000-pre, DEV: 2023.4.10097 ), if you have a header that is broken into two lines the chart will overrun the header.  

![Capture](https://github.com/datalust/seq-tickets/assets/8382528/828e5c5e-386d-4621-b8ae-f3a466f66e4c)

**Comment by liammclennan** [comment]
Hi @specialagentsoftware 

Thank you for the report. We will take a look.
